fleetspeak/src/server/admin/admin.go

```diff
@@ -249,3 +249,17 @@ func (s adminServer) BlacklistClient(ctx context.Context, req *spb.BlacklistClie
 	}
 	return &fspb.EmptyMessage{}, nil
 }
+
+func (s adminServer) FetchClientResourceUsageRecords(ctx context.Context, req *spb.FetchClientResourceUsageRecordsRequest) (*spb.FetchClientResourceUsageRecordsResponse, error) {
+	clientID, idErr := common.BytesToClientID(req.ClientId)
+	if idErr != nil {
+		return nil, idErr
+	}
+	records, dbErr := s.store.FetchResourceUsageRecords(ctx, clientID, int(req.Limit)) // todo: support generic table
```

Comment:
I'm not sure what this todo means.

Reply:
Remnant from the method I added and removed. Thanks for the catch!